endpoints/openrtb2/amp_auction.go

```diff
@@ -162,17 +163,28 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 			}
 		}
 	}
+	// Extract any errors
+	var extResponse openrtb_ext.ExtBidResponse
+	var respErrors map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderError
+	eRErr := json.Unmarshal(response.Ext, &extResponse)
+	if eRErr != nil {
+		glog.Errorf("AMP response: failed to unpack OpenRTB response.ext: %v", eRErr)
```

Comment:
I don't think we need to `glog` this... but I do think we should append it to the response errors under the `prebid` key so the publisher knows an error occurred, even if they don't know exactly what it is.